css/index.css

```diff
@@ -28,9 +28,32 @@ article, aside, details, figcaption, figure,
 footer, header, hgroup, menu, nav, section {
 	display: block;
 }
```

Comment:
Nice work splitting the CSS into separate files to keep the project more organized.